com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -916,7 +909,11 @@ private bool IsRenderPassEnabled(ScriptableRenderPass renderPass)
             // Track CPU only as GPU markers for this scope were "too noisy".
             using (new ProfilingScope(null, Profiling.RenderPass.configure))
             {
-                renderPass.Configure(cmd, cameraData.cameraTargetDescriptor);
+                if (IsRenderPassEnabled(renderPass) && cameraData.cameraType == CameraType.Game)
```

Comment:
`CameraType.VR` is also "game" camera.

Reply:
as discussed with the VR team, before enabling VR support a VR pass is needed to fix trunk issues and add URP support. It will be enabled only on mobile first (we have no means to test XR/VR ourselves)

Reply:
Ok. Sounds good.